README.md

```diff
@@ -4,41 +4,40 @@
 
 [Ably](https://ably.com/) is the platform that powers realtime experiences at scale, including live chat, data broadcast, notifications, audience engagement and collaboration.
 
-Multiplayer Collaboration enables virtual teams to work together efficiently by having the contextual awareness of who else is currently with them on the app, where exactly they are and the updates they are making. With the Ably Spaces SDK, we are on a mission to make it easy for developers to add Multiplayer Collaboration features to any web application, in a few lines of code.
+Multiplayer Collaboration enables virtual teams to work together efficiently by having the contextual awareness of who else is currently with them on the app, where exactly they are and the updates they are making. With the **Ably Collaborative Spaces SDK**, we are on a mission to make it easy for developers to add Multiplayer Collaboration features to any web application, in a few lines of code.
 
 ![Multiplayer Collaboration Demo](https://user-images.githubusercontent.com/5900152/225328262-2b63bb49-57a0-4f40-b78e-b87565f4c98c.png)
 
 > **Note**
-> The **Ably Spaces SDK** is a work in progress. It is currently meant for early adopters to trial and test out while some features are still under development. You can try it out with the usage instructions below.
+> The **Ably Collaborative Spaces** SDK is a work in progress and is currently meant for early adopters to trial and test out while some features are still under development.
 
 > If you are interested in seeing new API features that are still under development, [sign up for exclusive early access](https://go.ably.com/spaces-early-access). If you have any feedback or feature requests, [please fill out this form](https://go.ably.com/spaces-feedback) or reach out to us at [beta@ably.com](mailto:beta@ably.com).
 
 ## Overview
 
-The Ably Spaces SDK provides purpose-built APIs to enable multiplayer collaboration in any web application. You can set up a collaborative "space" and subscribe to different kinds of updates to see who else is in the space with you, their live location within the app and any updates they are making.
+Built as an extension to the [existing Ably-JS SDK](https://github.com/ably/ably-js), the Ably Collaborative Spaces SDK enables multiplayer collaboration in any web application through the creation of spaces. 
 
-It is built as an extension to our [existing JS SDK](https://github.com/ably/ably-js), so you'll need an Ably JS client to be able to use this library. The usage instructions will help you set that up.
+A space is a virtual collaborative location consisting of users and storage with persistent presence, enabling shared realtime digital experiences such as chat, reactions and collaborative editing. You can set up a collaborative space and subscribe to different kinds of updates to see who else is in the space with you, their live location within the application and any updates they are making.
```

Comment:
the terms "storage" and "chat" etc shouldn't be here, they incorrectly represent this library. Let's discuss this on Slack.